### ISSUE TITLE
Uneven table rows are not accessible to screen readers

### ISSUE THREAD

**Issue by molaachir** [dbeaver/dbeaver#5579]
When executing a query, and using Ctrl+6 to switch to the results grid, I cannot read the uneven table rows of my result. Only the even table rows are accessible. I use Windows 10, DBeaver 6.0.1, and jaws

**Comment by uslss** [comment]
cannot reproduce an issue.
if it is still actual for you - feel free to reopen the ticket
